_tools/generate_repo.py

```diff
@@ -170,22 +167,24 @@ def _generate_zip_file(self, path, version, addonid):
 
             for root, dirs, files in os.walk(path + os.path.sep):
 
-                if ignored_dirs:
-                    for ignored_dir in ignored_dirs:
-                        if ignored_dir in dirs:
-                            dirs.remove(ignored_dir)
+                for ignored_dir in ignored_dirs:
+                    if ignored_dir in dirs:
+                        dirs.remove(ignored_dir)
 
-                if ignored_files:
-                    for ignored_file in ignored_files:
-                        if ignored_file in files:
-                            files.remove(ignored_file)
+                for ignored_file in ignored_files:
+                    if ignored_file in files:
+                        files.remove(ignored_file)
```

Comment:
Will fail if ignored_files is None

Reply:
but it is a list defined on top of the script

Reply:
https://github.com/Twilight0/Repository-Bootstrapper/blob/master/_tools/generate_repo.py#L59

Reply:
Same thing applies here as the above comment.